.github/workflows/playwright.yml

```diff
@@ -1,43 +1,61 @@
 name: Playwright Tests
 
-on: [push, pull_request]
+on: workflow_call
 
 jobs:
-  test:
+  acceptance:
     strategy:
+      fail-fast: false
       matrix:
+        os: [windows-latest, ubuntu-latest, macos-latest]
         node-version: [17.x, 18.x, 19.x]
-    timeout-minutes: 60
-    runs-on: macos-latest
+    runs-on: ${{ matrix.os }}
+    defaults:
+      run:
+        working-directory: ./playground/locations-site
     steps:
-      - name: "Checkout"
-        uses: actions/checkout@v3
-
-      - name: Install pnpm
-        uses: pnpm/action-setup@v2.2.4
-
-      - name: Build with NodeJS ${{ matrix.node-version }}
-        uses: actions/setup-node@v3
+      - uses: actions/checkout@v3
+      - uses: pnpm/action-setup@v2.2.4
+      - uses: actions/setup-node@v3
         with:
           node-version: ${{ matrix.node-version }}
           cache: "pnpm"
+      - uses: actions/download-artifact@v3
+        with:
+          name: build-output
+          path: packages/pages/dist
 
-      - name: Install deps and build
-        run: pnpm i && pnpm run build
-
-      - name: Install again to link deps
-        run: pnpm i
+      # It's important to not run the build for @yext/pages on Windows.
+      # Currently the build does not work (on Windows) and will replace the
+      # downloaded build artifact without any warnings/errors.
+      - run: pnpm i --ignore-scripts
+      - run: ls ./node_modules/@yext/pages/dist/bin
```

Comment:
Is this output useful to have?

Reply:
I copy pasted from the older acceptance test workflow and missed this back then, will remove!